lib/bundler/cli/doctor.rb

```diff
@@ -101,25 +101,25 @@ def check_home_permissions
     end
 
     def check_for_files_not_owned_by_current_user_but_still_rw
-      return unless any_files_not_owned_by_current_user_but_still_rw?
+      return unless files_not_owned_by_current_user_but_still_rw.any?
       Bundler.ui.warn "Files exist in Bundler home that are owned by another " \
-        "user, but are stil readable/writable"
+        "user, but are stil readable/writable. These files are: #{files_not_owned_by_current_user_but_still_rw.join("\n")}"
```

Comment:
maybe `These files are:\n  - #{files.join("\n  - ")}` so theres some more visual differentiation?

Right now, this is the output I get:

```
Files exist in Bundler home that are not readable/writable to the current user. These files are: /Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/CLAide-7f26d3d016ef/.git/objects/pack/pack-37fe8b0ecb7fb4bb549140a193f9b3e5f57bd982.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/CLAide-7f26d3d016ef/.git/objects/pack/pack-37fe8b0ecb7fb4bb549140a193f9b3e5f57bd982.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Core-df21f20a2825/.git/objects/pack/pack-6a6bf20937cabbe439abad31192fb692d6585ae9.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Core-df21f20a2825/.git/objects/pack/pack-6a6bf20937cabbe439abad31192fb692d6585ae9.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Core-df6877912172/.git/objects/pack/pack-2c1d1f2113e0f0d1aa8cc9bbf30f63fc862dc08f.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Core-df6877912172/.git/objects/pack/pack-2c1d1f2113e0f0d1aa8cc9bbf30f63fc862dc08f.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Molinillo-b17c754d2b93/.git/objects/pack/pack-3d267bb97f882693d85a6e1b2851f665109fae4e.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Molinillo-b17c754d2b93/.git/objects/pack/pack-3d267bb97f882693d85a6e1b2851f665109fae4e.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Molinillo-b17c754d2b93/.git/objects/pack/pack-933914d968c3dd6509e6a5566f17d21f147f12f8.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Molinillo-b17c754d2b93/.git/objects/pack/pack-933914d968c3dd6509e6a5566f17d21f147f12f8.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Molinillo-c2f655cd6978/.git/objects/pack/pack-d63e0ec18a4aac8667d7c0bfc3deac537ad1f9f0.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Molinillo-c2f655cd6978/.git/objects/pack/pack-d63e0ec18a4aac8667d7c0bfc3deac537ad1f9f0.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Nanaimo-c9b1cf573452/.git/objects/pack/pack-20dc04ae125dbd210206bbb95986cf199c46d0e9.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Nanaimo-c9b1cf573452/.git/objects/pack/pack-20dc04ae125dbd210206bbb95986cf199c46d0e9.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Xcodeproj-4279382c6003/.git/objects/pack/pack-5584e2008093ac49ed59feb9082ee93cd70f0cc1.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Xcodeproj-4279382c6003/.git/objects/pack/pack-5584e2008093ac49ed59feb9082ee93cd70f0cc1.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Xcodeproj-d9be15fd7ac9/.git/objects/pack/pack-e4ebf9f8c265c099ac1869bd0b4c5ea222053a54.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/Xcodeproj-d9be15fd7ac9/.git/objects/pack/pack-e4ebf9f8c265c099ac1869bd0b4c5ea222053a54.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-deintegrate-3735087ac0d5/.git/objects/pack/pack-a08336c992731d0640ff4d3f6adb02131d2a8d5a.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-deintegrate-3735087ac0d5/.git/objects/pack/pack-a08336c992731d0640ff4d3f6adb02131d2a8d5a.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-downloader-74c8d27df0bf/.git/objects/pack/pack-7da53b6d4c74635672bcb2c8455171f14c79bdeb.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-downloader-74c8d27df0bf/.git/objects/pack/pack-7da53b6d4c74635672bcb2c8455171f14c79bdeb.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-plugins-c222b56ed644/.git/objects/pack/pack-23621f86d62c2e1c0722acb0f5621fac307f383b.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-plugins-c222b56ed644/.git/objects/pack/pack-23621f86d62c2e1c0722acb0f5621fac307f383b.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-search-eb60c856fc55/.git/objects/pack/pack-a86b94fa0bd854a521d7c6ab65bd67a27f9645d1.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-search-eb60c856fc55/.git/objects/pack/pack-a86b94fa0bd854a521d7c6ab65bd67a27f9645d1.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-stats-5eaccbc74899/.git/objects/pack/pack-4745e20dba5624f24613c734914364f70a54d5b3.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-stats-5eaccbc74899/.git/objects/pack/pack-4745e20dba5624f24613c734914364f70a54d5b3.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-trunk-649ccab360b3/.git/objects/pack/pack-a247d31191008395fabda8c36ac8e3ddbbd106d7.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-trunk-649ccab360b3/.git/objects/pack/pack-a247d31191008395fabda8c36ac8e3ddbbd106d7.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-trunk-a3d9e816e04b/.git/objects/pack/pack-cd3508e822e9ee6bf974849b8194a8db6222088e.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-trunk-a3d9e816e04b/.git/objects/pack/pack-cd3508e822e9ee6bf974849b8194a8db6222088e.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-try-2c0840a6ba64/.git/objects/pack/pack-7189861dc396805fd212e6aaee59813717ac8d4b.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/cocoapods-try-2c0840a6ba64/.git/objects/pack/pack-7189861dc396805fd212e6aaee59813717ac8d4b.pack
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/json-a9588bc4334c/.git/objects/pack/pack-4da8020924009c5291978ebe90f221cb3d33943f.idx
/Users/segiddins/Development/Rainforest/CocoaPods/.bundle/ruby/2.4.0/bundler/gems/json-a9588bc4334c/.git/objects/pack/pack-4da8020924009c5291978ebe90f221cb3d33943f.pack
```